### PR TITLE
[service.subtitles.opensubtitles-com] 1.0.4

### PR DESCRIPTION
**Pull request by opensubtitlesdev**
### Description

Multiple fixes on file path, language submitted by user "[pannal](https://github.com/pannal)" [opensubtitlesdev/service.subtitles.opensubtitles-com](https://github.com/opensubtitlesdev/service.subtitles.opensubtitles-com/pull/)
Improvements of the error messages for the user

### Checklist:

    * [x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project.

    * [ x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document

    * [ x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :

    * Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.

    * [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.

    * Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API

    * [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.

    * This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.

    * Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by opensubtitlesdev** [comment]
spending so much time answering support tickets for bugs solved by this release... please merge it...

File: service.subtitles.opensubtitles-com/resources/lib/os/provider.py [hunk 1]
```diff
@@ -69,9 +69,12 @@ def __init__(self, api_key, username, password):
         self.username = username
         self.password = password
 
-        self.request_headers = {"Api-Key": self.api_key, "User-Agent": "Opensubtitles.com Kodi plugin v1.0.3" ,"Content-Type": CONTENT_TYPE, "Accept": CONTENT_TYPE}
+        if not self.username or not self.password:
+            print(f"Username: {self.username}, Password: {self.password}")
```
Comment (romanvm) [inline service.subtitles.opensubtitles-com/resources/lib/os/provider.py:73]: Please use `xbmc.log()` instead

**Review by romanvm — CHANGES_REQUESTED** [review]

**Comment by romanvm** [comment]
Please fix one minor issue

File: service.subtitles.opensubtitles-com/resources/lib/os/provider.py (hunk 1 above)
Comment (opensubtitlesdev) [reply service.subtitles.opensubtitles-com/resources/lib/os/provider.py]: ok it's changed

**Comment by opensubtitlesdev** [comment]
ok, fixed, added a last minute little change in the data collection that caused errors with streamed movies 
https://github.com/opensubtitlesdev/service.subtitles.opensubtitles-com/pull/20

**Pull request merged**